managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -580,14 +585,6 @@ private synchronized void internalAsyncAddEntry(OpAddEntry addOperation) {
                 log.debug("[{}] Queue addEntry request", name);
             }
         } else if (state == State.ClosedLedger) {
-            long now = clock.millis();
-            if (now < lastLedgerCreationFailureTimestamp + WaitTimeAfterLedgerCreationFailureMs) {
```

Comment:
`WaitTimeAfterLedgerCreationFailureMs` can be removed as a constant

Reply:
will do